streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordQueue.java

```diff
@@ -59,11 +62,12 @@
         this.partition = partition;
         this.fifoQueue = new ArrayDeque<>();
         this.timestampExtractor = timestampExtractor;
+        this.skippedRecordsRateSensor = processorContext.metrics().taskLevelSensor(SKIPPED_RECORDS, processorContext.taskId().toString(), Sensor.RecordingLevel.INFO);
```

Comment:
We need to maintain a task-level skipped-records sensors since places like this are not correlated to any node-level sensors.